scripts/all-tests.sh

```diff
@@ -13,7 +13,40 @@ set -e
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=4175
 export JSHINT_THRESHOLD=7550
-export SAFELINT_THRESHOLD=2700
+export SAFELINT_THRESHOLDS='
```

Comment:
I was more-simply thinking of making this .sh file more readable. Maybe if the variable was set in a file, then that is the thing that would be touched whenever a given threshold needs to be adjusted. Not a must...just thinking it through.